src/inspected-window-helpers/overlay-helpers.js

```diff
@@ -130,9 +130,10 @@ export function setupOverlayHelpers() {
   }
 
   function getColorFromString(string, opacity = 0.4) {
+    const cleanStr = string.replace(/[^0-9a-z]/gi, "");
```

Comment:
This regex doesn't keep capitalized letters, and also doesn't keep non-ascii characters that might be used by those who speak other languages.

We should at least do `0-9a-zA-Z`, but are there any other characters we should allow?

Reply:
The `i` flag means its case insensitive, and does keep capitalized letters. 

```js
"@Some-org/module".replace(/[^0-9a-z]/gi, "") // returns "Someorgmodule"
```

I'm not sure how parseInt handles non-ascii characters but I vote for solving that when it comes up.

Reply:
Let me take that back. Now that we're sure that we have at least a few users in China (quiankun) maybe it should be solved now. I'll investigate.

Reply:
Perhaps using [toascii](https://github.com/zipavlin/toascii-js) to replace non-ascii in a determinate way, and that way the colors output are also consistent.

Reply:
I like the idea of replacing the nonascii characters when possible with a fallback.

Reply:
Or maybe just converting the ascii chars to charCode, which always return an integer and then everything is good to go. https://stackoverflow.com/questions/94037/convert-character-to-ascii-code-in-javascript

Mostly just writing down ideas before getting to it later tonight.

Reply:
ah i missed the `i` flag on the regex. Yeah this is fine then - if we can handle other characters that is good too. but this looks good to me